atc/worker_artifact.go

```diff
@@ -0,0 +1,8 @@
+package atc
+
+type WorkerArtifact struct {
+	ID        int    `json:"id"`
+	Name      string `json:"name"`
+	BuildID   int    `json:"build_id"`
+	CreatedAt int64  `json:"created_at"`
```

Comment:
Not actually sure what format we typically use for timestamps in the API, but we should check this to make sure we're consistent.

Reply:
from what i saw we aren't consistent lol